### DISCUSSION TITLE
CAN YOU GUYS HELP ME [ please :D ]

### DISCUSSION THREAD

**Discussion by Thejedi45** [farag2/Sophia-Script-for-Windows#397] [General]
https://imgur.com/a/XKGl0TZ

i dont know how to remove this error ....

![2022-10-21 00_30_56-Administrateur _ Windows PowerShell](https://user-images.githubusercontent.com/116308263/197071700-640d56a4-c6da-45e4-a699-547eec0e7469.png)

**Comment by farag2** [comment]
Look at the folder you're calling sophia.ps1: you're in Module!

**Reply by Thejedi45** [reply]
> Look at the folder you're calling sophia.ps1: you're in Module!

![2022-10-21 01_58_03-Resident Evil Showcase de DINGUE🚨 Resident Evil 4 s'annonce FOU 🔥 - YouTube — ](https://user-images.githubusercontent.com/116308263/197081191-873e2bcc-e1a3-4411-ad38-b15213c34614.png)

**Reply by Thejedi45** [reply]
so ?

**Comment by farag2** [comment]
Lol. Now you're calling the PowerShell 7 version of the script via PowerShell 5.1. :D

**Comment by farag2** [comment]
Download the script for your version! Your PowerShell is 5.1, you downloaded the version with PowerShell 7 in the name. It was 2 am: O couldn't reply fast. :)

**Reply by Thejedi45** [reply]
alright im gonna try this

**Comment by farag2** [comment]
The console even shows you the error 😸

**Comment by farag2** [comment]
Is it now working for you?)

**Reply by Thejedi45** [reply]
still not :(

**Reply by farag2** [reply]
Why?.. ping me in the Telegram group: https://t.me/sophia_chat
